config/routes.rb

```diff
@@ -57,11 +57,29 @@
     end
   end
 
+  # to be removed and below used when migrating to contentful
+  static_regexp = %r{
+    accessibility-statement|
+    cookie-policy|
+    new-registration|
+    other-problems-signing-in|
+    privacy-policy|
+    terms-and-conditions|
+    sitemap|
+    settings/cookies|
+    whats-new
+  }x
+  get '/:id', to: 'static#show', id: static_regexp, as: :static
+
+  constraints(ENV.fetch('CONTENTFUL', false) == true) do
```

Comment:
@agoldstone93 https://ey-recovery-pr-495.london.cloudapps.digital/promotional-materials looks good. Once merged though it'll break if we don't correct the envs passed in to terraform. In `main` it would look to `test` but be missed in the route because the feature flag was missing. I think we should keep `dev` pointing at `test` so we just need to make `CONTENTFUL: true` a default. @ebrett do you agree?

Reply:
Yes, with this going in, CONTENTFUL will need to be on for everything.